wordpress-importer.php

```diff
@@ -787,6 +787,7 @@ function process_posts() {
 						if ( isset( $inserted_comments[$comment['comment_parent']] ) )
 							$comment['comment_parent'] = $inserted_comments[$comment['comment_parent']];
 						$comment = wp_filter_comment( $comment );
+						$comment = wp_slash( $comment );
```

Comment:
The data should get slashed before calling `wp_filter_comment()`. This would make it consistent with current behaviour in core:

* https://github.com/WordPress/WordPress/blob/e357195ce303017d517aff944644a7a1232926f7/wp-includes/rest-api/endpoints/class-wp-rest-comments-controller.php#L593
* https://github.com/WordPress/WordPress/blob/76f44d242bb87ed47281aeb0750b39c8a4b00571/wp-includes/comment.php#L3095